cypress/support/pages/home.pageObject.js

```diff
@@ -7,10 +7,63 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
+  get settingsLink() {
+    return cy.getByDataCy('settings-link');
+  }
+
+  get newArticleLink() {
+    return cy.getByDataCy('new-article-link');
+  }
+
+  get homePageUrl() {
+    return cy.url();
+  }
+
+  get followBtn() {
+    return cy.getByDataCy('follow-user-btn')
+  }
+
+  get articleTitle() {
+    return cy.getByDataCy('article-title-link');
+  }
+
   assertHeaderContainUsername(username) {
     this.usernameLink
       .should('contain', username);
   }
+
+  clickOnSettingsLink() {
+    this.settingsLink
+      .click();
+  }
+
+  clickOnNewArticleLink(){
+    this.newArticleLink
+      .click();
+  }
+  verifyUrl() {
+    this.homePageUrl
+      .should('equal', url);
+  }
+
+  clickOnFollowBtn() {
+    this.followBtn
+      .click();
+  }
+
+  verifyFollow() {
+    this.followBtn
+      .should('contain', 'Unfollow');
+  }
+  verifyUnfollow() {
+    this.followBtn
+      .should('contain', 'Follow');
+  }
+
+  verifyAbsentArticle(title) {
+    this.articleTitle
+      .should('not.be.visible', title);
```

Comment:
The `verifyAbsentArticle` method uses an incorrect assertion. Instead of checking if the article title is 'not.be.visible', consider using 'should('not.exist')' to verify that the element is not present in the DOM.